### DISCUSSION TITLE
Start using biodiverse

### DISCUSSION THREAD

**Discussion by Yaselda** [shawnlaffan/biodiverse#943] [General]
Can someone please help me get started with Biodiverse?

**Comment by shawnlaffan** [comment]
Hello Yaselda, 

The quick start guide is the place to be begin: https://github.com/shawnlaffan/biodiverse/wiki/Downloads#quick-start-guide

There are also blog posts describing other functionality, although these are ordered by time and not topic.
http://biodiverse-analysis-software.blogspot.com/

Shawn.

**Reply by Yaselda** [reply]
Thank you, I will try it
